go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/hanzoai/gochimp3
+module github.com/tw1nk/gochimp3
```

Comment:
Was this supposed to be merged, @zeekay @tw1nk?

Reply:
Thanks for catching that!

Reply:
Sorry about that, haven't used GitHub for a couple of years and thought it didn't update pull requests unless you actively did an update on the webpage.